src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateCoverageComponents.java

```diff
@@ -80,6 +81,9 @@
 
     @Override
     protected void runPipeline(final JavaSparkContext ctx) {
+        if (! new HDF5Library().load(null)){
+            throw new UserException.HardwareFeatureException("Cannot load the required HDF5 library");
```

Comment:
Should these warnings be more verbose?  ex.  "HDF5 is supported on OSX and x86-64, if you are not running on one of these platforms you'll need to provide your own native hdf5 binaries by specifying....."

(is it possible to specify a different binary than we package?)